### ISSUE TITLE
node 12.19.0: Cannot find module '../lib/utils/unsupported.js'

### ISSUE THREAD

**Issue by rifler** [Schniz/fnm#283]
while calling `npm --version`

```
internal/modules/cjs/loader.js:834
  throw err;
  ^

Error: Cannot find module '../lib/utils/unsupported.js'
Require stack:
- /home/sandbox/.fnm/node-versions/v12.19.0/installation/bin/npm
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:831:15)
    at Function.Module._load (internal/modules/cjs/loader.js:687:27)
    at Module.require (internal/modules/cjs/loader.js:903:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at /home/sandbox/.fnm/node-versions/v12.19.0/installation/bin/npm:19:21
    at Object.<anonymous> (/home/sandbox/.fnm/node-versions/v12.19.0/installation/bin/npm:155:3)
    at Module._compile (internal/modules/cjs/loader.js:1015:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1035:10)
    at Module.load (internal/modules/cjs/loader.js:879:32)
    at Function.Module._load (internal/modules/cjs/loader.js:724:14) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ '/home/sandbox/.fnm/node-versions/v12.19.0/installation/bin/npm' ]
}
```

**Comment by Schniz** [comment]
I think I know what the issue is. I'm going to push a fix soon. Thanks!

**Comment by rifler** [comment]
this issue appeared today

**Comment by rifler** [comment]
@Schniz maybe there is a way to install specific version of fnm?

**Comment by Schniz** [comment]
All releases are in the release page. The installation script also supports a `--release VERSION` argument. But the fix is on its way. I reproduced it in a test and it seems to pass now.

**Comment by Schniz** [comment]
You can see why it happened (and to look at the test) here: https://github.com/Schniz/fnm/pull/284

**Comment by Schniz** [comment]
Fixed in v1.22.2
thanks!
